tests/e2e/matmul/CMakeLists.txt

```diff
@@ -976,6 +976,32 @@ iree_generated_e2e_matmul_test(
     "noriscv"
 )
 
+iree_generated_e2e_matmul_test(
+  NAME
+    e2e_matmul_rocm_f16_large_cdna3_matrixcore
```

Comment:
Is this running on CI anywhere? Or just for local use at the moment? I do want to set up `test_gpu_nvidia` and `test_gpu_amd` jobs once we have enough test runners.

We'll basically have a matrix around this to run tests on different machines: https://github.com/openxla/iree/blob/5d8907e82fc1eb741a4d4d27f5cae865323fd1d7/.github/workflows/ci.yml#L268-L388

(sort of like the `cross_compile_and_test` matrix, ... I need to take a closer look at that to see if it should exist separately or fold into that too)

Reply:
Local for now to make it easy to debug matmul correctness issues. But yes!!! I'd certainly want continous CI testing for it.